rt/transports/http-netty/netty-server/src/test/java/org/apache/cxf/transport/http/netty/server/integration/NettyServerTest.java

```diff
@@ -90,7 +90,8 @@ public void testGetWsdl() throws Exception {
         ByteArrayOutputStream bos = new ByteArrayOutputStream();
         IOUtils.copyAndCloseInput(in, bos);
         String result = bos.toString();
-        assertTrue("Expect the SOAPService", result.indexOf("<service name=\"SOAPService\">") > 0);
+        assertTrue("Expect the SOAPService", result.indexOf("<service name=\"SOAPService\">") > 0
```

Comment:
This might be an issue I believe, CXF should return consistent response (either qualified or unqualified namespaces), the test should not try to adapt to that.